proxy.go

```diff
@@ -158,7 +167,8 @@ func (ph ProxyHandler) proxyRequest(w http.ResponseWriter, req *http.Request, au
 			req.Body = ioutil.NopCloser(rd)
 			resp, err = auth.do(req, ph.transport)
 			if err != nil {
-				http.Error(w, err.Error(), http.StatusInternalServerError)
+				log.Printf("[%d] Error forwarding request (with auth): %v", id, err)
+				w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Unfortunately github does not allow me to comment on lines outside the diff context.

13 lines below this you are logging an error with the id, but you're pulling it out of the context. You can use `id` there now since you have it.

Also in other places you converted logging errors with `%v` - that error log still has `%s`.

Reply:
Done and done